spec/helpers/application_helper_spec.rb

```diff
@@ -41,28 +41,28 @@
     end
   end
 
-  describe "#remove_breadcrumbs" do
+  describe "#show_breadcrumbs?" do
     describe "when is_landing_page? is true" do
       let(:content_item) { ContentItem.new({ "schema_name" => "landing_page" }) }
 
       it "removes breadcrumbs" do
-        expect(remove_breadcrumbs(content_item)).to be(true)
+        expect(show_breadcrumbs?(content_item)).to be(false)
       end
     end
 
     describe "when is_landing_page? is false" do
       let(:content_item) { ContentItem.new({ "schema_name" => "a_different_page" }) }
 
       it "does not remove breadcrumbs" do
```

Comment:
And rename this test to "it shows breadcrumbs" etc

Reply:
Done 👍